MapleServer2/PacketHandlers/PacketHandler.cs

```diff
@@ -17,5 +17,10 @@ public static void LogUnknownMode(Enum mode)
         {
             Console.WriteLine("New Unknown " + mode.GetType().Name + ": 0x" + mode.ToString("X"));
         }
+        
+        public static void LogPacket(PacketReader packet)
+        {
+            Console.WriteLine("Unknown packet: " + packet.ToString());
```

Comment:
We should already log all packets automatically

Reply:
Not sure why this was marked as resolved?